lib/storefront/paypal.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="286px"
+	 height="76.2px" viewBox="0 0 286 76.2" style="enable-background:new 0 0 286 76.2;" xml:space="preserve">
+<title>Paypal</title>
+<desc>Paypal Logo</desc>
```

Comment:
Lol no they're the same except one has color! Which is not helpful for someone who can't see it anyway.